tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/AlfrescoStackInitializer.java

```diff
@@ -55,10 +74,18 @@ public void initialize(ConfigurableApplicationContext configurableApplicationCon
 
         liveIndexer = createLiveIndexingContainer(env);
 
-        startOrFail(elasticsearch, postgres, activemq, transformCore, transformRouter, liveIndexer, sfs);
+        startOrFail(elasticsearch);
+
+        startOrFail(postgres, activemq, sfs);
+
+        startOrFail(transformCore, transformRouter);
```

Comment:
Do we need the transformers for this test?

Reply:
Yes, we need transformers for some of the tests.